### PR TITLE
[cli][certs rm] (Major) Bump to v5 certs API

### PR DESCRIPTION
**Pull request by mglagola**
Bumps certs API from `/v3` to  `/v5`.

_Removing auto generated certs will be blocked in `/v5`_

### CODE REVIEW

**Review by anatrajkovska — DISMISSED** [review]
Do we display an error if removing an auto-generated cert?

**Review by javivelasco — DISMISSED** [review]

File: packages/now-cli/src/util/certs/get-custom-certs-for-domain.ts [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-  output: Output,
+export async function getCustomCertsForDomain(
   client: Client,
   context: string,
   domain: string
 ) {
   try {
     const { certs } = await client.fetch<Response>(
-      `/v3/now/certs?${stringify({ domain })}`
+      `/v3/now/certs?${stringify({ domain })}&custom=true`
```
Comment (anatrajkovska) [inline packages/now-cli/src/util/certs/get-custom-certs-for-domain.ts]:
```suggestion
      `/v3/now/certs?${stringify({ domain, custom: true })}`
```

**Review by anatrajkovska — DISMISSED** [review]
LGTM.

Left one suggestion.

**Review by anatrajkovska — DISMISSED** [review]

**Review by TooTallNate — DISMISSED** [review]

File: packages/now-cli/src/util/certs/get-custom-certs-for-domain.ts [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
-  output: Output,
+export async function getCustomCertsForDomain(
   client: Client,
   context: string,
   domain: string
 ) {
   try {
     const { certs } = await client.fetch<Response>(
-      `/v3/now/certs?${stringify({ domain })}`
+      `/v3/now/certs?${stringify({ domain, custom: true })}`
```
Comment (styfle) [inline packages/now-cli/src/util/certs/get-custom-certs-for-domain.ts]:
```suggestion
      `/v5/now/certs?${stringify({ domain, custom: true })}`
```

Should this be `v5` too?

**Review by anatrajkovska — APPROVED** [review]

**Review by styfle — APPROVED** [review]

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**